spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/web/jersey/JerseyEndpointResourceFactory.java

```diff
@@ -111,7 +111,7 @@ private Resource createEndpointLinksResource(String endpointPath,
 	private static final class EndpointInvokingInflector
 			implements Inflector<ContainerRequestContext, Object> {
 
-		private static final List<Function<Object, Object>> bodyConverters;
+		private static final List<Function<Object, Object>> BODY_CONVERTERS;
```

Comment:
We don't consider this to be a constant so the previous capitalisation was correct. Can you revert this part of the change please?

Reply:
@wilkinsona Sure but I thought having both `static` and `final` is generally considered as a constant and  I don't get the difference from `ServletComponentRegisteringPostProcessor.HANDLERS`: https://github.com/spring-projects/spring-boot/blob/master/spring-boot/src/main/java/org/springframework/boot/web/servlet/ServletComponentRegisteringPostProcessor.java#L46

Could you explain a little bit more?

Reply:
While the list of body converters is unmodifiable, each of its entries may not be. Specifically, you could add something that implemented `Function<Object, Object>` to the list that could also be mutated.

Applying that reasoning, `ServletComponentRegisteringPostProcessor` is wrong and the field should be named `handlers` not `HANDLERS`.

Reply:
@wilkinsona You mean a collection can't be a constant unless its element class is `final` and all fields in its element class are immutable like `String`. Do I understand correctly?

Reply:
You do indeed

Reply:
@wilkinsona Thanks for the clarification 👍 

I can see a few places which are broken by the rule. Is it okay to handle them in this PR?

Reply:
Of course! That'd would be much appreciated